src/RA_Dlg_GameLibrary.cpp

```diff
@@ -475,7 +473,7 @@ BOOL Dlg_GameLibrary::LaunchSelected()
     const int nSel = ListView_GetSelectionMark(hList);
     if (nSel != -1)
     {
-        TCHAR buffer[1024];
+        TCHAR buffer[1024]{};
```

Comment:
Arrays changed like this were to address a "string may not be null-terminated" warning.